### PR TITLE
pull request

### PR DESCRIPTION
**Pull request by ifly6**
reformatted a lot of stuff, which are most of the changes, mostly because my pycharm is set to auto-reformat (also removed unused imports). somehow, my branch is now called email-backups but its in reference to your e-balloting. i don't really know why that happened. it certainly, however, is based on the e-balloting code rather than the email-backups. (it might be worthwhile in the future to merge the branches)

i fixed the lenientLate bug where mittab didn't create the key when it initialised a new tournament. also added documentation for the views and some of the models. there was also a bug in casting `datetime.now()` to string, that's been solved. did some changes to the import logic, but all minor.

### CODE REVIEW

**Comment by ifly6** [comment]
oh. that's why (looking at the history). i attempted to merge email-backups into eballots. then reverted it back, which changed the name of the thing and made the history iffy.

File: mittab/apps/tab/models.py [hunk 1]
```diff
@@ -43,43 +54,65 @@ def delete(self):
...
 
 class Debater(models.Model):
-    name = models.CharField(max_length=30, unique = True)
-    #team_set is created by Team in the ManyToMany
-    #team = models.ForeignKey('Team')
-    #0 = Varsity, 1 = Novice
+    """
+    Contains information with regard to the debaters at the tournament.
+    """
+    # TODO it's honestly probably better to make a super-class containing the provider, phone, and name information.
```
Comment (BenMusch) [inline mittab/apps/tab/models.py:65]: +1

File: mittab/apps/tab/models.py [hunk 2]
```diff
@@ -43,43 +54,65 @@ def delete(self):
...
             raise Exception("Debater on teams: %s" % ([t.name for t in teams]))
 
+
 class Team(models.Model):
-    name = models.CharField(max_length=30, unique = True)
+    """
+    Contains information with regard to the team itself. Contains the team name, the team school, the debaters in that
+    team, the team seed, and the information on whetehr that team is checked in or not.
+    """
+    alphanumeric = RegexValidator(r'^[0-9a-zA-Z ]*$', 'Only alphanumeric characters are allowed.')
```
Comment (BenMusch) [inline mittab/apps/tab/models.py:105]: Should also allow hyphenated names

File: mittab/libs/tab_logic.py [hunk 3]
```diff
@@ -437,105 +456,131 @@ def team_wins_by_forfeit():
...
 
-""" Ranks """
-@cache()
+
+"""
+Team Ranks Methods
+"""
+
+
+# @cache()
```
Comment (BenMusch) [inline mittab/libs/tab_logic.py:519]: What's the reasoning for removing the caching?

**Review by BenMusch — COMMENTED** [review]
Looks good to me overall, gonna merge and try to get this running on the server to make sure everything works

**Pull request merged**